### ISSUE TITLE
A11y_SQLMigrationControlPlane_  AssessmentResultsAndRecommendations _View/SelectButton_ScreenReader: Proper accessible name is not announced by the screen reader for the link "SQL Server Agent differences in azure SQL Managed Instance.

### ISSUE THREAD

**Issue by BhanuPrakashGit** [microsoft/azuredatastudio#20651]
“[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

GitHubTags:#A11yMAS;#A11yTCS;#AzureDataStudioextension'AzureSQLMigration'-Mac-Sept2022;#SQL Migration Control Plane;#A11ySev2;#WCAG1.3.1;#DesktopApp;#Win11;#ScreenReader;#Benchmark;#VoiceOver;#Mac

### Environment Details:
Azure Data Studio: Version: 1.39.1
Operating System: Mac Version 12.5.1
Screen reader: VoiceOver

### Steps to Reproduce:

1. Install Azure data studio and open
2. Tab till Extension press enter key and install Azure SQL Migration 
3. Tab till Respective server and right click
4. Tab till manage and press enter key
5. 5Tab till Azure SQL Migration and press enter key
6. Tab till migrate to azure SQL and press enter key
7. Saved session page will open 
8. Select start new session and give next
9. Databases for assessment page will open 
10. Select the respective databases and give next 
11. Assessment results and recommendations page will open
12. Select Azure SQL managed Instance /Virtual Machines from Azure SQL Target
13. Tab till View/Select button, assessment results for '<server-name>' dialog will open
14. Observe that Proper accessible name is not announced by the screen reader for the link "SQL Server Agent differences in azure SQL Managed Instance".

### Actual:
Proper accessible name is not announced by the screen reader for the link "SQL Server Agent differences in azure SQL Managed Instance". It is only announcing "more info" link.

### Expected:
Proper accessible name should be announced by the screen reader for the link "SQL Server Agent differences in azure SQL Managed Instance. It should announce like "more info, SQL Server Agent differences in azure SQL Managed Instance, link.

### User Impact:
Screen reader users will get not proper info related to that link.
[Screen Recording 2022-09-14 at 3.44.31 PM.zip](https://github.com/microsoft/azuredatastudio/files/9631835/Screen.Recording.2022-09-14.at.3.44.31.PM.zip)

**Comment by aasimkhan30** [comment]
Can you please add the right aria-label to the more info links?

**Comment by BhanuPrakashGit** [comment]
The expected aria-label should be "**SQL Server Agent differences in azure SQL Managed Instance**".

**Comment by brian-harris** [comment]
this issue is fixed in PR #20265

**Comment by brian-harris** [comment]
this issue was fixed as part of this PR:
[20265](https://github.com/microsoft/azuredatastudio/pull/20265)

**Comment by Charles-Gagnon** [comment]
@brian-harris Please don't close a11y issues yourself, the a11y needs to verify and close them - so you should @-mention them to let them know they need to recheck.

(if there isn't a response in a week after doing that then we can close them ourselves though)

**Comment by Charles-Gagnon** [comment]
@BhanuPrakashGit Please validate that the issue is fixed using the latest insiders build

**Comment by BhanuPrakashGit** [comment]
@Charles-Gagnon Tried to update the application but getting below error. Could you please tell me from where we can download the latest insiders build or what action should I take here? Thanks

![MicrosoftTeams-image (13)](https://user-images.githubusercontent.com/104351566/199402243-45ef0089-e335-400f-877e-c3713b0d9f56.png)

**Comment by Charles-Gagnon** [comment]
Did you click the link in the notification? That should provide information on what to do when you see this error. 

Here's the direct link to the Insiders download though if you want to just start fresh with the latest version : https://github.com/Microsoft/azuredatastudio#try-out-the-latest-insiders-build-from-main

**Comment by BhanuPrakashGit** [comment]
Verified the bug in the env: 
Azure Data Studio
Version: 1.39.1
Bug is fixed and does not repro. Hence closing the bug. 
PFA for reference
[Screen Recording 2022-11-28 at 11.38.15 AM.zip](https://github.com/microsoft/azuredatastudio/files/10103610/Screen.Recording.2022-11-28.at.11.38.15.AM.zip)

**Comment by InduPriya1805** [comment]
Githubtags:  #AzureDataStudioextensionAzureSQLMigration-Mac-Sept2022;

**Comment by BhanuPrakashGit** [comment]
Githubtags: #Closed;
